tests/testthat/test-cov-struct.R

```diff
@@ -132,3 +132,21 @@ test_that("print.cov_struct pretty-formats covariance structure object", {
     "`VIS IT`, VISIT | GROUP / SUBJECT"
   )
 })
+
```

Comment:
please also add an integration test that resembles Yoni's original problem, showing that it is fixed now. thanks

Reply:
added now